include/nbl/builtin/hlsl/math/quadrature/gauss_legendre/gauss_legendre.hlsl

```diff
@@ -37,6 +37,7 @@ struct GaussLegendreIntegration
 
 #define float_t float32_t
 #include <nbl/builtin/hlsl/math/quadrature/gauss_legendre/impl.hlsl>
+#undef float_t
```

Comment:
btw they fixed the bug in DXC (well they added a capability trimming pass), you can always declare both `floa64_t` and `float32_t`